erigon-lib/kv/tables.go

```diff
@@ -352,17 +352,18 @@ const (
 	StateCommitment = "StateCommitment"
 
 	// BOR
-	BorReceipts       = "BorReceipt"
-	BorFinality       = "BorFinality"
-	BorTxLookup       = "BlockBorTransactionLookup" // transaction_hash -> block_num_u64
-	BorSeparate       = "BorSeparate"               // persisted snapshots of the Validator Sets, with their proposer priorities
-	BorEvents         = "BorEvents"                 // event_id -> event_payload
-	BorEventNums      = "BorEventNums"              // block_num -> event_id (first event_id in that block)
-	BorSpans          = "BorSpans"                  // span_id -> span (in JSON encoding)
-	BorMilestones     = "BorMilestones"             // milestone_id -> milestone (in JSON encoding)
-	BorMilestoneEnds  = "BorMilestoneEnds"          // start block_num -> milestone_id (first block of milestone)
-	BorCheckpoints    = "BorCheckpoints"            // checkpoint_id -> checkpoint (in JSON encoding)
-	BorCheckpointEnds = "BorCheckpointEnds"         // start block_num -> checkpoint_id (first block of checkpoint)
+	BorReceipts           = "BorReceipt"
```

Comment:
Why not keep the same variable name as the string here?

Reply:
not sure I follow - what do you mean?

Reply:
Oh are you talking about why it is missing the "s" in BorReceipt? 
I don't know. Probably typo. It is not easy to change it because it is a DB table name that is used.
Also it is not related to this PR :)